add examples for vertx-amqp-bridge module

gemmellr opened the pull request.

cescoffier commented:
Thanks, 

I will have a look this weekend.

cescoffier commented on amqp-bridge-examples/src/main/java/io/vertx/example/amqp/Sender.java:
```diff
@@ -0,0 +1,60 @@
+  }
+
+  // Convenience method so you can run it in your IDE
+  public static void main(String[] args) {
```
IT's generally the first method in the class.

cescoffier commented:
Looks good except a small comment.
Could you also add a reference to these example in the root README file.

gemmellr commented:
I have updated the README (oops, forgot all about it), and moved the main methods for consistency (force of habbit, I generally put mains at the end as they are often the least interesting bit of the example to me)

cescoffier commented:
Great, thanks !
